src/instrumentation/version.ts

```diff
@@ -0,0 +1,19 @@
+import { isVersionMetadata } from './env.js'
+
+export function versionAttributes(env: unknown): Record<string, string | undefined> {
+	const attributes = {} as Record<string, string | undefined>
+
+	if (typeof env === 'object' && env !== null) {
+		for (const [binding, data] of Object.entries(env)) {
+			if (isVersionMetadata(data)) {
+				attributes['cf.gradual_rollouts.binding'] = binding
+				attributes['cf.gradual_rollouts.id'] = data.id
+				attributes['cf.gradual_rollouts.tag'] = data.tag
```

Comment:
You can have version ID/Tag without gradual rollouts, I think? Should this be something like `cf.script_version.id` or something?